app/javascript/graphql-explorer/graphql-fetcher.js

```diff
@@ -0,0 +1,24 @@
+// Defines a GraphQL fetcher using the fetch API. You're not required to
+// use fetch, and could instead implement graphQLFetcher however you like,
+// as long as it returns a Promise or Observable.
+export default function graphQLFetcher(graphQLParams) {
```

Comment:
Maybe the new ES6 Arrow function syntax here too?
Just the way you did it below in index.js